app/assets/javascripts/shares_editor.js

```diff
@@ -1,4 +1,4 @@
-const setupOnce = require('setup_once');
+let setupOnce = require('setup_once');
```

Comment:
No - I think I removed imports at one point and just added it back with `let`. I didn't see much of a risk of it getting modified. Would you like me to change it back?